_variables.tf

```diff
@@ -476,3 +476,8 @@ variable "ecs_service_capacity_provider_strategy" {
   description = "(Optional) The capacity provider strategy to use for the service. Can be one or more. These can be updated without destroying and recreating the service only if set to [] and not changing from 0 capacity_provider_strategy blocks to greater than 0, or vice versa."
   default     = [{}]
 }
+
+variable "alarm_ecs_running_tasks_threshold" {
+  default     = 1
```

Comment:
I think it should be 0 and if alarm_ecs_running_tasks_threshold > 1 the aws_cloudwatch_metric_alarm will be created

Reply:
Please check the description for more information.
https://github.com/DNXLabs/terraform-aws-ecs-app/blob/6785b0329b7481ce4e5a59e752da24cd83597acc/_variables.tf#L482

Reply:
fair enough,